target/linux/ath79/image/generic.mk

```diff
@@ -330,6 +330,17 @@ define Device/comfast_cf-wr650ac-v2
 endef
 TARGET_DEVICES += comfast_cf-wr650ac-v2
 
+define Device/comfast_cf-wr752ac-v1
+  SOC := qca9531
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-WR752AC
+  DEVICE_VARIANT := v1
+DEVICE_PACKAGES := kmod-leds-gpio kmod-usb2 kmod-ath10k-ct ath10k-firmware-qca9888-ct \
```

Comment:
Wrong indent, more than 80 characters per line.

Reply:
OK, such rules are new to me.